src/edu/washington/escience/myria/operator/CSVFileScanFragment.java

```diff
@@ -159,20 +164,21 @@ public CSVFileScanFragment(
 
     this.delimiter = MoreObjects.firstNonNull(delimiter, CSVFormat.DEFAULT.getDelimiter());
     this.quote = MoreObjects.firstNonNull(quote, CSVFormat.DEFAULT.getQuoteCharacter());
-    this.escape = escape != null ? escape : CSVFormat.DEFAULT.getEscapeCharacter();
+    this.escape = escape;
```

Comment:
Because the CSVFormat.DEFAULT for escape is null I believe

On Tue, Jul 26, 2016 at 1:41 PM, Tobin Baker <notifications@github.com>
wrote:

> In src/edu/washington/escience/myria/operator/CSVFileScanFragment.java
> <https://github.com/uwescience/myria/pull/822#discussion_r72332674>:
>
> > @@ -159,20 +164,21 @@ public CSVFileScanFragment(
> >
> >      this.delimiter = MoreObjects.firstNonNull(delimiter, CSVFormat.DEFAULT.getDelimiter());
> >      this.quote = MoreObjects.firstNonNull(quote, CSVFormat.DEFAULT.getQuoteCharacter());
> > -    this.escape = escape != null ? escape : CSVFormat.DEFAULT.getEscapeCharacter();
> > +    this.escape = escape;
>
> Why isn't this line just like the others?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/uwescience/myria/pull/822/files/065f1661b24e07587fe9ca6f887e26ba011c3c34..8dd5e3cfc34d0381f9222654c3bcfd76999f037b#r72332674>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADFus5c2AdVYRUV1RiLZqApyI1G5Wk4Lks5qZnEOgaJpZM4IMdpy>
> .
>